docs/one-click-integrations.md

```diff
@@ -10,13 +10,52 @@ To set up the PagerDuty integration:
 
 1.  On Pulse, click **Integrations** and select **PagerDuty**.
 
-    ![PagerDuty integratio](images/pagerduty.png)
+    ![PagerDuty integration](images/pagerduty.png)
 
 1.  Fill in the necessary details:
 
-    -    **PagerDuty subdomain:** Your custom PagerDuty subdomain. For example, `mysubdomain` if you use the URL `https://mysubdomain.pagerduty.com` to access PagerDuty.
-    -    **Pulse system name:** Name of the [system](https://docs.pulse.codacy.com/#before-you-begin) to associate with the incidents reported by this integration.
+    - **PagerDuty subdomain:** Your custom PagerDuty subdomain. For example, `mysubdomain` if you use the URL `https://mysubdomain.pagerduty.com` to access PagerDuty.
+    - **Pulse system name:** Name of the [system](https://docs.pulse.codacy.com/#before-you-begin) to associate with the incidents reported by this integration.
 
 1.  Follow the instructions on the Pulse UI to create a new webhook on PagerDuty using the provided URL. For example:
 
     ![PagerDuty webhook](images/pagerduty-webhook.png)
+
+## GitHub
+
+Pulse integrates directly with GitHub to receive data about deployments and changes.
+
+!!! important
+    Consider the following before using the GitHub integration:
+
+    -   The integration obtains information about deployments from Git tags that follow the [SemVer](https://semver.org) convention, excluding pre-release versions.
+
+    -   The integration uses the creation date of [annotated tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging#_annotated_tags) as the deployment date.
+    
+        However, since [lightweight tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging#_lightweight_tags) don't have this information, in this case the deployment date is the timestamp when the integration receives the webhook call. Since webhook calls can be delayed, the deployment date on Pulse might be imprecise, impacting the metric **Lead time for changes**.
+
+    -   The integration obtains the set of changes that belong to a deployment from the list of commits between the tag of that deployment and the previous tag.
+    
+        Because of this, the integration doesn't consider the first SemVer tag in the repository as a deployment because it's not possible to calculate the list of changes for that deployment.
```

Comment:
@afsmeira and @rtfpessoa, I added the [detail about the integration "discarding" the first SemVer tag](https://codacy.atlassian.net/browse/PUL-227?focusedCommentId=45157).

Reply:
possible is a strong word. We could simply consider all the commits before that tag. I guess this was more of a decision.
Maybe @DReigada can help with the wording.

Reply:
You're right. Perhaps:

> Because of this, the integration discards the first SemVer tag in the repository because there isn't a sure way to calculate the list of changes for that deployment.

Reply:
I think it would be nice to refer the "previous tag", since that is the actual problem

> Because of this, the integration discards the first SemVer tag in the repository since there is no previous tag to compare with.